mods/ts/weapons/ballisticweapons.yaml

```diff
@@ -110,6 +110,13 @@ RPGTower:
 		Explosions: large_explosion
 		ImpactSounds: expnew06.aud
 
+155mmDeployCheck: ## DummyWeapon
```

Comment:
This is quite a large hack. I think you have to solve it differently. Maybe try `IOverrideAutoTarget` instead?